lib/config/options/index.ts

```diff
@@ -3120,6 +3120,13 @@ const options: RenovateOptions[] = [
     default: false,
     globalOnly: true,
   },
+  {
+    name: 'skipArtifactUpdating',
+    description: 'If set, artifact updating will be skipped.',
```

Comment:
this will be the first line in docs above, so it should integrate with the text above